webapp-django/crashstats/signature/jinja2/signature/signature_correlations.html

```diff
@@ -1,6 +1,6 @@
 <div id="correlations-wrapper"
      data-default-channel="{{ channel }}"
-     data-default-product="{{ product }}">
+     data-default-product="{{ product_name }}">
```

Comment:
Correlations is so weird in the Crash Stats code right now. This is one of the things that should eventually get rewritten using product details.